OpenWeatherAPI.Tests/APITests.cs

```diff
@@ -0,0 +1,27 @@
+using Newtonsoft.Json;
+using System.Diagnostics;
+using Xunit;
+
+namespace OpenWeatherAPI.Tests
+{
+	/// <summary>
+	/// Tests <see cref="API"/>
+	/// </summary>
+	public class APITests
+	{
+		[Fact()]
+		public void QueryTest_Success()
+		{
+			//Arrange
+			var api = new API("a94a22bf61976a8575caacbb14401dfa");
```

Comment:
Is this your API key, or is this dummy data?

Reply:
Oops, that is mine, I'have invalidated it at openweather, and I'll remove it.

Reply:
..done